<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CProjectOverviewDialog</class>
 <widget class="QDialog" name="CProjectOverviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="OpenProjectButton">
       <property name="text">
        <string>Open Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ExportGameButton">
       <property name="text">
        <string>Export Game</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ViewResourcesButton">
     <property name="text">
      <string>View Resources</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="WorldsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Worlds</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="WorldsList"/>
      </item>
      <item>
       <widget class="QPushButton" name="LoadWorldButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="AreasGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Areas</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="AreaComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LaunchEditorButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Launch World Editor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
